modules/statistics/AnalyticsAdapter.js

```diff
@@ -15,63 +16,40 @@ AnalyticsAdapter.eventsQueue = [];
  * Sends analytics event.
  * @param action
  * @param data
```

Comment:
So now "data" became some structure. I would love to see some documentation on what it is.